src/main/java/com/tasker/server/boards/rest/BoardCreationController.java

```diff
@@ -16,9 +20,33 @@
     private BoardService boardService;
 
     @PostMapping("/boards")
-    ResponseEntity<String> createBoard(@Valid @RequestBody BoardCreationRequest request) {
-        boardService.create(request);
+    ResponseEntity<BoardCreatedResponse> createBoard(@Valid @RequestBody BoardCreationRequest request) {
 
-        return ResponseEntity.ok("Board Created");
+        Board board = new Board();
+
+        board.setId(UUID.randomUUID());
+        board.setCreatedAt(new Date());
+        board.setTitle(request.getTitle());
+        board.setDescription(request.getDescription());
+
+        boardService.create(board);
+
+        return new ResponseEntity<>(new BoardCreatedResponse(board.getId().toString()), HttpStatus.CREATED);
```

Comment:
what is the need for toString() method here, board.getId() returns String , doesn't it?

Reply:
ok, I confused board.getId() with BoardCreatedResponse class' getId() method.